src/components/course/routes/tests/ExternalCourseEnrollment.test.jsx

```diff
@@ -136,6 +147,36 @@ describe('ExternalCourseEnrollment', () => {
     expect(screen.getByTestId('user-enrollment-form')).toBeInTheDocument();
   });
 
+  it.each([
+    // The course exists and is "available", but non-redeemable via subsidy.
+    {
+      contentKey: mockCourseRunKey,
+      hasSuccessfulRedemption: false,
```

Comment:
[nit/non-blocking] Since `hasSuccessfulRedemption` and the redirect to `enroll/success` is handled in the loader, I am wondering if theres some way to handle this test case if hasSuccessfulRedemption is true

Reply:
Hm, so I traced a hooks and found no usage of `hasSuccessfulRedemption` within this component.  So I'm pretty sure in order to test the outcome of toggling this we'd need to add tests to the loader tests.  Does that answer your question?